packages/models/src/event.js

```diff
@@ -161,10 +161,14 @@ export default class Event {
 
   get requestPath() {
     if (this.httpServerRequest) {
-      return (
-        this.httpServerRequest.normalized_path_info ||
-        this.httpServerRequest.path_info
-      );
+      const normalizedPathInfo = this.httpServerRequest.normalized_path_info
+        ? this.httpServerRequest.normalized_path_info.toString()
+        : '';
```

Comment:
@dustinbyrne Sasha says this is an empty array sometimes. Is there a way to normalize this even farther upstream?

Reply:
Events are mutated during normalization here: https://github.com/applandinc/appmap-js/blob/main/packages/models/src/appMapBuilder/index.js#L90-L114

This would be the best place to fix up any standardization issues.

Reply:
Fair enough. I moved that code to AppMap builder's `normalize()` method.  https://github.com/applandinc/appmap-js/pull/558/commits/7a74df0093d1279613b45096451b2012a6477c93